src/Monitor/Monitor/Alerts/AddAzureRmMetricAlertRuleV2Command.cs

```diff
@@ -95,7 +96,7 @@ public class AddAzureRmMetricAlertRuleV2Command : ManagementCmdletBase
         [Parameter(Mandatory = true, ValueFromPipeline = true,ValueFromPipelineByPropertyName = true, HelpMessage = "The condition for rule")]
         [ValidateNotNullOrEmpty]
         [Alias("Criteria")]
-        public List<PSMetricCriteria> Condition { get; set; }
+        public List<IPSMultiMetricCriteria> Condition { get; set; }
```

Comment:
Potential Breaking Change: Change of parameter type (unless the type contains all the properties the previous parameter type contained)